README.md

```diff
@@ -236,6 +236,17 @@ As mentioned previously the policy file must adhere to a naming convention
 See additional property requirements in Query policies and the aforementioned sample Github repository.
 
 
+#### Resource Notification Policies
+
+Resource Notification policies are useful when you wants to send notification for a resource(Org, Apps, Services) to the emailid's configured on the resources as labels. 
```

Comment:
when you want to generate a report containing resources of a particular type and send that report to email recipients identified by one or more label values ascribed to the those resources. 

For example, you might notify primary and/or secondary owners of a collection of a resources.

Reply:
Done